epoch/api/contract_api_usage.md

```diff
@@ -10,6 +10,34 @@ There are two basic types of API calls, off-chain operations and on-chain transa
 For an up to date list of all HTTP API endpoints and their arguments see
 [Epoch HTTP API](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/master/apps/aehttp/priv/swagger.json)
 
+## Sophia calldata creation
+
+A number of functions described below create calldata for contract call or
+contract create transactions ([code/encode-calldata](#encode-call-data),
+[code/call](#call), [create/compute](#contract-create),
+[call/compute](#contract-call)). These support two ways of specifying the
+function and arguments to the call:
+- **Unchecked (legacy)**: Function name (except for `create/compute`) and arguments
+  as a Sophia constant tuple. No checks are made to ensure that the given
+  arguments match what the contracts expects.
+- **Checked**: An argument `call` containing Sophia source code for a contract
+  snippet containing (at least) a prototype for the function to be called and a
+  special function `__call()` whose body is a single call to this function. This
```

Comment:
As mentioned in [sibling code review](https://github.com/aeternity/epoch/pull/1709#pullrequestreview-169862829) - not sure if __call shall be documented in __call as not recommented name for public function.